tools/bpf2c/bpf2c_dll.c

```diff
@@ -33,8 +33,8 @@ division_by_zero(uint32_t address)
     fprintf(stderr, "Divide by zero at address %d\n", address);
 }
 
-#define FIND_METADATA_ENTRTY(NAME, X) \
-    if (std::string(NAME) == #X)      \
+#define FIND_METADATA_ENTRY(NAME, X) \
```

Comment:
Is this macro used anywhere? I could not find it being used. If it is not used, I think we can remove this altogether.